WBooks/New Group/Library/LibraryViewController.swift

```diff
@@ -7,3 +7,56 @@
 //
 
 import Foundation
+import UIKit
+import WolmoCore
+
+final class LibraryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
It would be nice if you use extensions for delegate and data source, in this way you can have a more ordered code and separate different responsibilities